### PR TITLE
Collijk/feature/model 6 treatment effect

### PR DESCRIPTION
**Pull request by collijk**

### CODE REVIEW

File: src/vivarium_csu_sanofi_multiple_myeloma/components/treatment.py [hunk 1]
```diff
@@ -41,6 +42,79 @@ def make_treatment_coverage(year, scenario):
     return coverage
 
 
+def make_progression_hazard_ratio():
+    # TODO: Get a distribution from researchers and sample.
```
Comment (yongqx) [inline src/vivarium_csu_sanofi_multiple_myeloma/components/treatment.py:46]: Yes, we can convert mean (lb, ub) to a distribution for progression HR and mortality HR.

**Review by yongqx — APPROVED** [review]

File: src/vivarium_csu_sanofi_multiple_myeloma/components/treatment.py [hunk 2]
```diff
@@ -41,6 +42,79 @@ def make_treatment_coverage(year, scenario):
...
+    first_line_isa = 0.429  # lower = 0.368, upper = 0.495
+    first_line_dara = 0.429  # lower = 0.368, upper = 0.495
+    first_line_residual = 1.00581  # lower = 1.0051, upper = 1.0064
+
+    later_line_isa = 0.530  # lower = 0.356, upper = 0.803
+    later_line_isa_retreat = 0.765  # lower = 0.678, upper = 0.902
+    later_line_dara = 0.217  # lower = 0.203, upper = 0.231
```
Comment (aflaxman) [inline src/vivarium_csu_sanofi_multiple_myeloma/components/treatment.py:53]: @alibow can you check this 0.217 value again? I am still surprised that it is so low.

File: src/vivarium_csu_sanofi_multiple_myeloma/components/treatment.py [hunk 3]
```diff
@@ -41,6 +42,79 @@ def make_treatment_coverage(year, scenario):
...
+    hazard_ratio['year_end'] = 2100
+    return hazard_ratio
+
+def make_mortality_hazard_ratio():
+    # TODO: Get a distribution from researchers and sample.
+    first_line_isa = 0.760  # lower = 0.645, upper = 0.895
+    first_line_dara = 0.760  # lower = 0.645, upper = 0.895
+    first_line_residual = 1.0024  # lower = 1.0011, upper = 1.0036
+
+    later_line_isa = 1.159  # lower = 1.044, upper = 1.185
```
Comment (aflaxman) [inline src/vivarium_csu_sanofi_multiple_myeloma/components/treatment.py:87]: @alibow I'm also surprised that this is > 1.0!

File: src/vivarium_csu_sanofi_multiple_myeloma/components/treatment.py [hunk 4]
```diff
@@ -41,6 +42,79 @@ def make_treatment_coverage(year, scenario):
...
+    hazard_ratio.loc[(models.SUSCEPTIBLE_STATE_NAME, models.TREATMENTS.no_treatment, False)] = 1.0
+
+    line = models.MULTIPLE_MYELOMA_1_STATE_NAME
+    hazard_ratio.loc[(line, models.TREATMENTS.isatuxamib, False)] = first_line_isa
+    hazard_ratio.loc[(line, models.TREATMENTS.daratumamab, False)] = first_line_dara
+    hazard_ratio.loc[(line, models.TREATMENTS.residual, False)] = first_line_residual
+
+    for line in TREATMENT_LINES.tolist()[1:]:
+        hazard_ratio.loc[(line, models.TREATMENTS.isatuxamib, True)] = later_line_isa
+        hazard_ratio.loc[(line, models.TREATMENTS.isatuxamib, False)] = later_line_isa_retreat
```
Comment (aflaxman) [inline src/vivarium_csu_sanofi_multiple_myeloma/components/treatment.py:105]: @collijk : Are you sure the True/False values here mean what you intend them to mean?  They seem reversed from the line-1 meaning.

**Review by aflaxman — APPROVED** [review]

File: src/vivarium_csu_sanofi_multiple_myeloma/components/treatment.py (hunk 3 above)
Comment (collijk) [reply src/vivarium_csu_sanofi_multiple_myeloma/components/treatment.py:87]: me too!

File: src/vivarium_csu_sanofi_multiple_myeloma/components/treatment.py (hunk 4 above)
Comment (collijk) [reply src/vivarium_csu_sanofi_multiple_myeloma/components/treatment.py:105]: You are correct, they are switched.

**Review by mattkappel — APPROVED** [review]

**Pull request merged**